components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -1831,40 +2000,37 @@ String[] filterScopeValidators(OAuthConsumerAppDTO application) throws IdentityO
     /**
      * Get the IdToken Encryption Method registered by the user and filter the allowed one.
      *
-     * @param application Application user have registered
+     * @param encryptionMethod Encryption method sent in the registration request.
      * @return idTokenEncryptionMethod
      * @throws IdentityOAuthAdminException Identity OAuthAdmin exception.
      */
-    private String filterIdTokenEncryptionMethod(OAuthConsumerAppDTO application) throws IdentityOAuthAdminException {
+    private String filterEncryptionMethod(String encryptionMethod, String configName)
+            throws IdentityOAuthAdminException {
 
-        List<String> supportedIdTokenEncryptionMethods = OAuthServerConfiguration.getInstance()
-                .getSupportedIdTokenEncryptionMethods();
-        String idTokenEncryptionMethod = application.getIdTokenEncryptionMethod();
-        if (!supportedIdTokenEncryptionMethods.contains(idTokenEncryptionMethod)) {
-            String msg = String.format("'%s' IdToken Encryption Method is not allowed.", idTokenEncryptionMethod);
+        List<String> supportedIdTokenEncryptionMethods = IdentityUtil.getPropertyAsList(configName);
```

Comment:
Lets change the variable name `supportedIdTokenEncryptionMethods`

Reply:
changed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2178/commits/f6b06f0d8192508c55bf5ed2309a3732f2a16680